yarn-project/ivc-integration/src/wasm_client_ivc_integration.test.ts

```diff
@@ -1,13 +1,7 @@
-import { BB_RESULT, executeBbClientIvcProof, verifyClientIvcProof } from '@aztec/bb-prover';
-import { ClientIvcProof } from '@aztec/circuits.js';
 import { createDebugLogger } from '@aztec/foundation/log';
 
 import { jest } from '@jest/globals';
```

Comment:
There are now three tests that tests the same circuits: the native version, this 'wasm' version that uses bb.js as a library' and a browser version that executes through playwright. Prob we don't need all three, your call.